incubating/composition-editor/step.yaml

```diff
@@ -81,7 +81,7 @@ spec:
   stepsTemplate: |-
     composition-editor:
       name: composition-editor
-      image: quay.io/codefreshplugins/composition-editor:1.0.0
+      image: quay.io/codefreshplugins/composition-editor:1.0.2
```

Comment:
Image does not exist
```
$ docker pull quay.io/codefreshplugins/composition-editor:1.0.2
Error response from daemon: manifest for quay.io/codefreshplugins/composition-editor:1.0.2 not found: manifest unknown: manifest unknown
```

Reply:
The image is not built until it is merged to master branch.